samples/components/dialogs/new_text_field/connectionBuilder.js

```diff
@@ -2,20 +2,23 @@
 {
     var params = {};
 
-    params["SERVER"] = attr2["server"];
-    params["PORT"] = attr2["port"];
-    params["DATABASE"] = attr2["dbname"];
-    params["UID"] = attr2["username"];
-    params["PWD"] = attr2["password"];
+    params["SERVER"] = attr2[connectionHelper.attributeServer];
+    params["PORT"] = attr2[connectionHelper.attributePort];
+    params["DATABASE"] = attr2[connectionHelper.attributeDatabase];
+    params["UID"] = attr2[connectionHelper.attributeUsername];
+    params["PWD"] = attr2[connectionHelper.attributePassword];
     params["BOOLSASCHAR"] = "0";
     params["LFCONVERSION"] = "0";
     params["UseDeclareFetch"] = "1";
     params["Fetch"] = "2526";
 
     //Pull out the attributes from the connection dialog.
-    //You can preserve these attributes by adding service and attributeWarehouse to the required attribute list.
+    //You can preserve these attributes by adding service, vendor attributes and attributeWarehouse to the required attribute list.
     params["WH_TEST"] = attr2[connectionHelper.attributeWarehouse];
```

Comment:
remove `params["WH_TEST"] = attr2[connectionHelper.attributeWarehouse];` now that this sample no longer showing that in the dialog